content/en/docs/setup/windows/intro-windows-in-kubernetes.md

```diff
@@ -0,0 +1,400 @@
+* The `--kubelet-reserve`, `--system-reserve` , and `--eviction-hard` flags update [NodeAllocatable](/docs/tasks/administer-cluster/reserve-compute-resources/#node-allocatable)
+* Eviction by using `--enforce-node-allocable` is not implemented
+* Eviction by using `--eviction-hard` and `--eviction-soft` are not implemented
+* A kubelet running on a Windows node does not have memory
+  restrictions. `--kubelet-reserve` and `--system-reserve` do not set limits on
+  kubelet or processes running on the host. This means kubelet or a process on the host
+  could cause memory resource starvation outside the node-allocatable and scheduler.
+* The `MemoryPressure` Condition is not implemented
+* The kubelet does not take OOM eviction actions
+
+### API compatibility {#api}
+
+There are no differences in how most of the Kubernetes APIs work for Windows. The
+subtleties around what's different come down to differences in the OS and container
+runtime. In certain situations, some properties on workload resources were designed
+under the assumption that they would be implemented on Linux, and fail to run on Windows.
+
+At a high level, these OS concepts are different:
+
+* Identity - Linux uses userID (UID) and groupID (GID) which
+  are represented as integer types. User and group names
+  are not canonical - they are just an alias in `/etc/groups`
+  or `/etc/passwd` back to UID+GID. Windows uses a larger binary
+  [security identifier](https://docs.microsoft.com/en-us/windows/security/identity-protection/access-control/security-identifiers) (SID)
+  which is stored in the Windows Security Access Manager (SAM) database. This
+  database is not shared between the host and containers, or between containers.
+* File permissions - Windows uses an access control list based on (SIDs), whereas
+  POSIX systems such as Linux use a bitmask based on object permissions and UID+GID,
+  plus _optional_ access control lists.
+* File paths - the convention on Windows is to use `\` instead of `/`. The Go IO
+  libraries typically accept both and just make it work, but when you're setting a
+  path or command line that's interpreted inside a container, `\` may be needed.
+* Signals - Windows interactive apps handle termination differently, and can
+  implement one or more of these:
+  * A UI thread handles well-defined messages including `WM_CLOSE`.
+  * Console apps handle Ctrl-C or Ctrl-break using a Control Handler.
+  * Services register a Service Control Handler function that can accept
+    `SERVICE_CONTROL_STOP` control codes.
+
+Container exit codes follow the same convention where 0 is success, and nonzero is failure.
+The specific error codes may differ across Windows and Linux. However, exit codes
+passed from the Kubernetes components (kubelet, kube-proxy) are unchanged.
+
+##### Field compatibility for container specifications {#compatibility-v1-pod-spec-containers}
```

Comment:
The level 5 headings here should, I think, become level 4 headings.

Ideally: move these field compatibility details into a new Reference page, and add inbound hyperlinks from relevant pages.